frontend/packages/console-app/src/components/tour/AdminGuidedTourBanner.tsx

```diff
@@ -0,0 +1,517 @@
+import { SVGProps } from 'react';
+
+const AdminGuidedTourBanner = (props: SVGProps<SVGSVGElement>) => (
+  <svg
+    id="Layer_1"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlnsXlink="http://www.w3.org/1999/xlink"
+    viewBox="0 0 1080 1080"
+    {...props}
+    width="100%"
+  >
+    <defs>
+      <style>
+        {
+          '\n      .st0, .st1, .st2, .st3, .st4, .st5, .st6, .st7, .st8 {\n        fill: none;\n      }\n\n      .st9 {\n        fill: #a60000;\n      }\n\n      .st10 {\n        fill: #92c5f9;\n      }\n\n      .st1, .st2, .st4, .st5, .st6, .st7, .st8 {\n        stroke-linecap: round;\n      }\n\n      .st1, .st2, .st5, .st6, .st7, .st8, .st11 {\n        stroke-miterlimit: 10;\n      }\n\n      .st1, .st3, .st4 {\n        stroke: #000;\n      }\n\n      .st1, .st3, .st4, .st8 {\n        stroke-width: 2.16px;\n      }\n\n      .st12 {\n        fill: #dede55;\n      }\n\n      .st13 {\n        fill: #e0f0ff;\n      }\n\n      .st2, .st11 {\n        stroke: #b9dafc;\n        stroke-width: 3px;\n      }\n\n      .st3, .st4 {\n        stroke-linejoin: round;\n      }\n\n      .st14 {\n        fill: #f56e6e;\n      }\n\n      .st5 {\n        stroke: #21134d;\n        stroke-width: 8.63px;\n      }\n\n      .st15 {\n        fill: #40251d;\n      }\n\n      .st16 {\n        clip-path: url(#clippath-1);\n      }\n\n      .st17 {\n        clip-path: url(#clippath-3);\n      }\n\n      .st18 {\n        clip-path: url(#clippath-4);\n      }\n\n      .st19 {\n        clip-path: url(#clippath-2);\n      }\n\n      .st20 {\n        clip-path: url(#clippath-7);\n      }\n\n      .st21 {\n        clip-path: url(#clippath-6);\n      }\n\n      .st22 {\n        clip-path: url(#clippath-8);\n      }\n\n      .st23 {\n        clip-path: url(#clippath-5);\n      }\n\n      .st24, .st11 {\n        fill: #fff;\n      }\n\n      .st6 {\n        stroke: #fff;\n      }\n\n      .st6, .st7 {\n        stroke-width: 4px;\n      }\n\n      .st25 {\n        fill: #b9dafc;\n      }\n\n      .st7 {\n        stroke: #e00;\n      }\n\n      .st26 {\n        fill: #724335;\n      }\n\n      .st27 {\n        fill: #21134d;\n      }\n\n      .st28 {\n        fill: #a0a;\n      }\n\n      .st29 {\n        fill: #4394e5;\n      }\n\n      .st8 {\n        stroke: #40251d;\n      }\n\n      .st30 {\n        clip-path: url(#clippath);\n      }\n\n      .st31 {\n        fill: #e00;\n      }\n    '
```

Comment:
can we clean this up a bit? perhaps by removing the newlines and whitespace, since it's not really helping readability